tests/test_cli.py

```diff
@@ -102,8 +102,8 @@ def test_cli_almanack_check(tmp_path):
             "Datetime:",
             "Almanack version:",
             "Target repository path:",
-            "The following Software Gardening Almanack metrics have failed:",
-            "Software Gardening Almanack score:",
+            "The following Software Gardening Almanack metrics",
+            "Software Gardening Almanack check ratio:",
```

Comment:
```suggestion
            "The following Software Gardening Almanack metrics have been evaluated",
            "Software Gardening Almanack check ratio:",
```
That felt like an incomplete sentence so I added a recommendation for what would sound good or neutral there.